samples/retry-policy/RetryPolicyExample.cs

```diff
@@ -0,0 +1,23 @@
+namespace MyNamespace;
+
+using Temporalio.Workflows;
+
+[Workflow]
+public class RetryPolicyExample
+{
+    [WorkflowRun]
+    public async Task<string> RunAsync(string name)
+    {
+        return await Workflow.ExecuteActivityAsync(
+            (MyActivities act) => act.SayHello(name),
+            new() { StartToCloseTimeout = TimeSpan.FromSeconds(90), // Schedule a retry if the Activity function doesn't return within 90 seconds
+            },
```

Comment:
Mentioned in other PR, but would recommend turning on analyzers for all projects and having CI confirm that all code is formatted (e.g. `dotnet format --verify-only`)